tests/scripts/all.sh

```diff
@@ -3327,7 +3327,10 @@ component_build_mingw () {
     make CC=i686-w64-mingw32-gcc AR=i686-w64-mingw32-ar LD=i686-w64-minggw32-ld CFLAGS='-Werror -Wall -Wextra' WINDOWS_BUILD=1 SHARED=1 lib programs
     make CC=i686-w64-mingw32-gcc AR=i686-w64-mingw32-ar LD=i686-w64-minggw32-ld CFLAGS='-Werror -Wall -Wextra' WINDOWS_BUILD=1 SHARED=1 tests
     make WINDOWS_BUILD=1 clean
-}
+
+    msg "build: Windows cross build - mingw64, make (Library only, AESNI intrinsics)" # ~ 30s
+    make CC=i686-w64-mingw32-gcc AR=i686-w64-mingw32-ar LD=i686-w64-minggw32-ld CFLAGS='-Werror -Wall -Wextra -maes -msse2 -mpclmul' WINDOWS_BUILD=1 lib
```

Comment:
So the only difference between this build line and 3320 is the addition of the command-line flags. It seems that there should be a difference in configuration too?

Reply:
`2.28` allows build `MBEDTLS_AESNI_C` even if there is no implementation usable. So line 3320 is actually build without `MBEDTLS_AESNI_C` because `MBEDTLS_AESNI_HAVE_CODE` is not defined in that case. It is not exactly same, but I need to say it should be very close to disabling `MBEDTLS_AESNI_C`.